src/_guides/libraries/library-tour.md

```diff
@@ -1035,6 +1035,38 @@ For more information, see
 [Exceptions](/guides/language/language-tour#exceptions)
 (in the language tour) and the [Exception API reference.][Exception]
 
+### Weak references and finalizers
+Dart is a [garbage-collected][] language,
+which means that any Dart object
+that isn't referenced by at least one other object
+will eventually be disposed by the garbage collector. 
+This default behavior may not be desireable in
+some scenarios involving native resources or 
+if the target object cannot be sub-classed.
```

Comment:
can't be subclassed? What is this referring to?

Reply:
I'm aware of some scenarios where an Expando / WeakMap is useful for adding properties to objects. Maybe this should be more specific...

Reply:
My understanding is that WeakMap is used for tools that work alongside an app

Reply:
That's a bit of a strange thing to say under `### Weak references and finalizers`. If you want to cover the use cases for Expandos as well, then maybe change the header to include expandos.

Reply:
I'll change this to "can't be modified"